src/sentry/static/sentry/app/views/projectAlertRules.jsx

```diff
@@ -56,12 +80,20 @@ const RuleRow = createReactClass({
   },
 
   render() {
-    let {orgId, projectId, data} = this.props;
-    let editLink = `/${orgId}/${projectId}/settings/alerts/rules/${data.id}/`;
+    let {data} = this.props;
+    let editLink = recreateRoute(`${data.id}/`, this.props);
+
     return (
-      <div className="box">
-        <div className="box-header">
-          <div className="pull-right">
+      <Panel>
+        <PanelHeader
+          css={{paddingTop: 5, paddingBottom: 5}}
+          isFlex
+          align="center"
+          justify="space-between"
+        >
+          <TextColorLink to={editLink}>{data.name}</TextColorLink>
+
+          <div>
             <Button style={{marginRight: 5}} size="small" to={editLink}>
               {t('Edit Rule')}
```

Comment:
It's being capitalised now since it's inside the panel header. Can we revert to normal casing?

Reply:
@ckj thoughts on this?

Reply:
Looks like it's addressed here https://github.com/getsentry/sentry/pull/7403/files#diff-f40c60afd931b143af44de4852062e33R12